### DISCUSSION TITLE
Unable to run mitmproxy in kali terminal

### DISCUSSION THREAD

**Discussion by v3cn4x00** [mitmproxy/mitmproxy#6289] [Q&A]
┌──(root㉿kali)-[~/BDFProxy]
└─# mitmproxy --version
Traceback (most recent call last):
  File "/usr/local/bin/mitmproxy", line 2, in <module>
    from libmproxy.main import mitmproxy
  File "/usr/local/lib/python3.11/dist-packages/libmproxy/main.py", line 52
    except ProxyServerError, v:
           ^^^^^^^^^^^^^^^^^^^
SyntaxError: multiple exception types must be parenthesized

Please help me with this problem

**Comment by ThomasBucaioni** [comment]
Maybe try the binary, it works on Kali in my VM: https://mitmproxy.org/

**Reply by v3cn4x00** [reply]
Ok will try

**Comment by Mahesh-adpl** [comment]
how to install the binary in kali?

**Reply by ThomasBucaioni** [reply]
Just run it, no need for install
